guest/testdata/cyclestate/main.go

```diff
@@ -109,10 +111,25 @@ func (statePlugin) Filter(state api.CycleState, pod proto.Pod, _ api.NodeInfo) (
 	return
 }
 
-func (statePlugin) PreScore(state api.CycleState, pod proto.Pod, _ proto.NodeList) *api.Status {
+func (statePlugin) PostFilter(state api.CycleState, pod proto.Pod, _ api.NodeToStatusMap) (nominatedNodeName string, nominatingMode api.NominatingMode, status *api.Status) {
 	if unsafe.Pointer(pod.Spec()) != unsafe.Pointer(podSpec) {
 		panic("didn't cache pod from filter")
 	}
+	mustNotScoreState(state)
+	if val, ok := state.Read(preFilterStateKey); !ok {
+		panic("didn't propagate state from pre-filter")
+	} else if _, ok = val.(preFilterStateVal)["filter"]; !ok {
+		panic("filter value lost propagating from post-filter")
+	} else {
+		val.(preFilterStateVal)["postfilter"] = struct{}{}
+	}
+	return
+}
+
+func (statePlugin) PreScore(state api.CycleState, pod proto.Pod, _ proto.NodeList) *api.Status {
+	if unsafe.Pointer(pod.Spec()) != unsafe.Pointer(podSpec) {
+		panic("didn't cache pod from post-filter")
```

Comment:
pre-filter

Reply:
Fixed. BTW prescore is after post-filter. So I thought it should be post-filter. Do I misunderstand something?

Reply:
After post-filter, no extension point is called.

The scheduling cycle is either
- prefilter → filter --(if no suitable node)--> post-filter → (owari)
- prefilter → filter → prescore → score → ...

Reply:
Also, this cache for pod should always be stored at pre-fiter.
https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/blob/6354c6d855a7149042809b6e881bb65d16c31a77/guest/testdata/cyclestate/main.go#L90

Reply:
> Also, this cache for pod should always be stored at pre-fiter.

Fixed it, but if I misunderstood, please let me know.